ts/components/cie/CieRequestAuthenticationOverlay.tsx

```diff
@@ -97,51 +90,18 @@ const CieWebView = (props: Props) => {
   const webView = createRef<WebView>();
   const { onSuccess } = props;
 
+  const handleOnError = React.useCallback(() => {
+    setInternalState(state => generateErrorState(state));
+  }, []);
+
   // Android CIE login flow is different from iOS.
   // On Android to be sure to regenerate a new crypto key,
   // we need to pass a new value to useLollipopLoginSource: loginUriRetry.
-  const { loginSource, regenerateLoginSource } =
-    useLollipopLoginSource(loginUri);
-  const [webviewSource, setWebviewSource] = React.useState<
-    WebViewSource | undefined
-  >(undefined);
-  const [lollipopCheckStatus, setLollipopCheckStatus] =
-    React.useState<LollipopCheckStatus>({ status: "none", url: O.none });
-
-  const verifyLollipop = React.useCallback(
-    (eventUrl: string, urlEncodedSamlRequest: string, publicKey: PublicKey) => {
-      setWebviewSource(undefined);
-      lollipopSamlVerify(
-        urlEncodedSamlRequest,
-        publicKey,
-        () => {
-          setLollipopCheckStatus({
-            status: "trusted",
-            url: O.some(eventUrl)
-          });
-          setWebviewSource({ uri: eventUrl });
-        },
-        () => {
-          setLollipopCheckStatus({
-            status: "untrusted",
-            url: O.some(eventUrl)
-          });
-          handleOnError();
-        }
-      );
-    },
-    []
-  );
-
-  const startLoginProcess = React.useCallback(() => {
-    if (loginSource.kind === "ready") {
-      setWebviewSource(loginSource.value);
-    }
-  }, [loginSource]);
-
-  useEffect(() => {
-    startLoginProcess();
-  }, [startLoginProcess]);
+  const {
+    retryLolliPoPLogin,
```

Comment:
I should stay consistent with the friendly term "lollipop" without any assumption about acronyms like PoP or POP as we did from the beginning. What do you think? This gives us also a better **camel case** naming when we use something related to this feature/functionality.

Reply:
Perfect, I do agree with you. Updated